django/db/backends/postgresql/client.py

```diff
@@ -22,13 +22,17 @@ def settings_to_cmd_args_env(cls, settings_dict, parameters):
         sslcert = options.get('sslcert')
         sslkey = options.get('sslkey')
 
+        if not dbname and not service:
+            # Connect to the default 'postgres' db.
+            dbname = 'postgres'
```

Comment:
Is this necessary? I'm sure that `psql` defaults to `postgres` if `dbname` is unspecified.

```suggestion
```

Reply:
As far as I'm aware it's necessary, e.g.
```
  -d, --dbname=DBNAME      database name to connect to (default: "felixx")
```

Reply:
Ah, I missed that because I looked in the man page, not at `--help` output, and forgot it defaults to the current user's username.

And, yes, we'd rather get a chance of connecting to a known, valid database rather than a failure to connect because the user's database doesn't exist (which is highly likely).